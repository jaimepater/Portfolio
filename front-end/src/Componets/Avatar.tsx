import * as React from 'react';
import styled from 'styled-components';
import imageJaime from '../assets/Jaime.png';


const Avatar = styled.div`
  background-image: url(${imageJaime});
  border-radius: 50%;  
  height: 100%;
  width: 100%;
  background-size: 100% 100%;
`;

export default Avatar;
