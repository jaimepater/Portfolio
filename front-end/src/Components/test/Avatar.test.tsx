import React from "react";
import { shallow } from 'enzyme';
import Avatar from "../Avatar";

describe('Avatar', () => {
  it('Renders Avatar', () => {
    const shallowComponent = shallow(<Avatar />);
    expect(shallowComponent).toMatchSnapshot();
  });
});