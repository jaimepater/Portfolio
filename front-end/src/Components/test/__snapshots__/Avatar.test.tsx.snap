// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar Renders Avatar 1`] = `
<div
  className="sc-AykKC bkEdCw"
/>
`;
